cibuildwheel/__main__.py

```diff
@@ -105,30 +105,8 @@ def main():
     # This needs to be passed on to the docker container in linux.py
     os.environ['CIBUILDWHEEL'] = '1'
 
-    try:
-        project_setup_py = os.path.join(project_dir, 'setup.py')
-        name_output = subprocess.check_output([sys.executable, project_setup_py, '--name'],
-                                              universal_newlines=True)
-        # the last line of output is the name
-        package_name = name_output.strip().splitlines()[-1]
-    except subprocess.CalledProcessError as err:
-        if not os.path.exists(project_setup_py):
```

Comment:
I'm thinking. While the whole of the try-except statement is not useful anymore, maybe this `if not os.path.exists(project_setup_py): ...` statement is useful? Failing early and giving the user a readable error message might be a good idea here, in case the project directory argument is wrong and doesn't contain a setup.py.

So I would suggest to actually keep those three lines and keep the rest deleted? @joerick, any thoughts?

(Sorry for changing my mind/not being more clear in the previous review :-/ )

Reply:
yeah, i'd agree, the test for setup.py is still useful, since there's no point in going to the trouble to e.g. downloading the docker images if there's no setup.py. So it would be good to keep that check.